### ISSUE TITLE
Time to fork?

### ISSUE THREAD

**Issue by tedivm** [thias/puppet-sysctl#64]
This module is pretty popular- but it also doesn't appear to work for Puppet5, and hasn't had a new upgrade in years. Further its main developer appears to have abandoned it.

In the past others have volunteered to help maintain it. @saz volunteered in #55, and the idea of transferring this to @voxpupili came up in #59.

**Comment by alexharv074** [comment]
@thias

**Comment by alexharv074** [comment]
I am currently being blocked on a project I'm doing, due to the module requiring an old version of Puppet Concat.

**Comment by hdep** [comment]
Yes I think this is needed. this module is usefull for the community !

**Comment by bschonec** [comment]
I managed to get in contact with Matthias today and he's willing to donate the module to the Voxpupuli community.  I'm not exactly sure how the process works.  Anyone?  Bueller?

**Comment by zilchms** [comment]
Just stumbled upon this. I can help you with the process. Or at least give some help and get the correct people into the loop :)
The process is documented at: https://voxpupuli.org/docs/migrate_module/
If you need anything more, let me know

**Comment by bschonec** [comment]
I asked the Voxpupuli folks about this a few months ago and someone told me that Voxpupuli already has a module that manages sysctl.  He said there wouldn't be a reason to maintain two modules.

**Comment by zilchms** [comment]
Ah alright then. Never mind :)

**Comment by bschonec** [comment]
I sent an email to the project owner asking if he'd transfer ownership to a 3rd party.  I'll update the issue if/when I hear back.

**Comment by bschonec** [comment]
@thias I sent another email to @thias to request Voxpupuli transfer.

**Comment by bschonec** [comment]
It looks like [puppet-augeasproviders_sysctl](https://github.com/voxpupuli/puppet-augeasproviders_sysctl) is Voxpupuli's sysctl management tool.

**Comment by saz** [comment]
There are some differences in how things are behaving and I'd still say, this module should be forked by voxpupuli.
